Source/Scene/Cesium3DTileset.js

```diff
@@ -518,6 +519,13 @@ define([
          */
         this.loadSiblings = defaultValue(options.loadSiblings, false);
 
+        /**
+         * The {@link ClippingPlanesCollection} used to selectively disable rendering the tileset.
+         *
+         * @type {ClippingPlanesCollection}
+         */
+        this.clippingPlanes = options.clippingPlanes;
```

Comment:
BUG:  I set this value to a `ClippingPlaneCollection` and it worked correctly.  Then I set this value to `undefined` but the tileset was still clipped